pulsar-io/elastic-search/src/main/java/org/apache/pulsar/io/elasticsearch/ElasticSearchConfig.java

```diff
@@ -251,6 +251,13 @@
     )
     private boolean copyKeyFields = false;
 
+    @FieldDoc(
+            required = false,
+            defaultValue = "true",
+            help = "If ignoreUnsupportedFields is true, unsupported AVRO fields are nullified and AVRO logical types are decoded as known AVRO types, otherwise it fails."
+    )
+    private boolean ignoreUnsupportedFields = true;
```

Comment:
please set this to "false" by default, otherwise users may experience "data loss" without knowing